src/usdb_syncer/notes_parser.py

```diff
@@ -384,6 +388,21 @@ def __str__(self) -> str:
     def artist_title_str(self) -> str:
         return f"{self.artist} - {self.title}"
 
+    def fix_apostrophes(self) -> None:
+        for key in (
+            "artist",
+            "title",
+            "language",
+            "genre",
+            "creator",
```

Comment:
Maybe we should skip this one? We can also skip `comment` since we don't use that field.

Reply:
You mean skip title, language, genre, creator and comment? But accents used as apostrophes are always wrong. They could be used in all those tags.

Reply:
No, sorry, I just meant creator. If someone chooses to be called `It´s me`, we probably want to reuse that name literallly, don't we?

Reply:
Ah, I see. Yes, maybe we want to take the creator's name literally, even though it hurts my eyes ;-)